CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Change Log
+
+## [v0.0.4](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.4) (2016-12-05)
+[Full Changelog](https://github.com/kabirbaidhya/keycode-js/compare/v0.0.3...v0.0.4)
+
+**Implemented enhancements:**
+
+- Add tests [\#1](https://github.com/kabirbaidhya/keycode-js/pull/1) [[enhancement](https://github.com/kabirbaidhya/keycode-js/labels/enhancement)] ([mesaugat](https://github.com/mesaugat))
+
+## [v0.0.3](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.3) (2016-10-28)
+[Full Changelog](https://github.com/kabirbaidhya/keycode-js/compare/v0.0.2...v0.0.3)
+
+## [v0.0.2](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.2) (2016-04-14)
+[Full Changelog](https://github.com/kabirbaidhya/keycode-js/compare/v0.0.2-0...v0.0.2)
+
+## [v0.0.2-0](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.2-0) (2016-04-05)
+[Full Changelog](https://github.com/kabirbaidhya/keycode-js/compare/v0.0.1-0...v0.0.2-0)
+
+## [v0.0.1-0](https://github.com/kabirbaidhya/keycode-js/tree/v0.0.1-0) (2016-04-04)
+
+
+\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```

Comment:
Same here single word.

Reply:
Not possible right now, unless [they](https://github.com/skywinder/Github-Changelog-Generator) make it so :smile: .